src/app/clusters/time-synchronization-server/time-synchronization-server.h

```diff
@@ -142,10 +142,11 @@ class TimeSynchronizationServer : public FabricTable::Delegate
     static TimeSynchronizationServer sTimeSyncInstance;
     TimeSyncEventFlag mEventFlag = TimeSyncEventFlag::kNone;
 #if TIME_SYNC_ENABLE_TSC_FEATURE
-    Platform::UniquePtr<app::ClusterStateCache> mAttributeCache;
-    Platform::UniquePtr<app::ReadClient> mReadClient;
     chip::Callback::Callback<chip::OnDeviceConnected> mOnDeviceConnectedCallback;
     chip::Callback::Callback<chip::OnDeviceConnectionFailure> mOnDeviceConnectionFailureCallback;
+    Attributes::UTCTime::TypeInfo::DecodableType mTrustedNodeUtcTime;
+    Attributes::Granularity::TypeInfo::DecodableType mTrustedNodeGranularity;
+    Platform::UniquePtr<app::ReadClient> mReadClient;
```

Comment:
Note: we could have a class that has mTrustedNodeUtcTime, mTrustedNodeGranularity, and ReadClient (by value, not pointer) members, then only need. the memory for the time and granularity while in the middle of the update.  Not sure how much that matters in practice, though.

Reply:
done.